client/dialogs/SmartIDProgress.cpp

```diff
@@ -166,6 +166,9 @@ SmartIDProgress::SmartIDProgress(QWidget *parent)
 			qCWarning(SIDLog) << reply->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt();
 			returnError(reply->readAll());
 			return;
+		case QNetworkReply::AuthenticationRequiredError:
+			returnError(tr("Check your Smart-ID service access settings."));
```

Comment:
use tr("Check your %1 service access settings.").arg("Smart-ID") for reusing single translation